opentelemetry-otlp/src/metric.rs

```diff
@@ -169,7 +169,7 @@ where
     ///
     /// [exporter-docs]: https://github.com/open-telemetry/opentelemetry-specification/blob/a1c13d59bb7d0fb086df2b3e1eaec9df9efef6cc/specification/metrics/sdk_exporters/otlp.md#additional-configuration
     pub fn with_delta_temporality(self) -> Self {
-        self.with_temporality_selector(DeltaTemporalitySelector)
+        self.with_temporality_selector(DeltaTemporalitySelector::new())
```

Comment:
the future of these methods in OTLP crate is unknown based on https://github.com/open-telemetry/opentelemetry-rust/pull/1811
However, this change itself is fine.